docs/collecting-data/collecting-from-own-applications/mobile-trackers/plugins/focal-meter/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Sending user identifier to Kantar FocalMeter"
+date: "2023-01-04"
+sidebar_position: 20
+---
+
+# Sending user identifier to Kantar FocalMeter
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+The tracker has the ability to send the user identifier (`userId` present in the session context) to a [Kantar FocalMeter](https://www.virtualmeter.co.uk/focalmeter) endpoint.
```

Comment:
Can we add like 1-2 sentences at the beginning about what a focalmeter is? In case people land on the page and are like, what is this, do I need to care about this

Reply:
Good point! Have added a bit more intro what focalmeter is.